Photons.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe we want to also include `project.xcworkspace` in our `.gitignore` 

If you agree with this and will be adding it, might as well add `.DS_Store` while you're at it 🙃